README.md

```diff
@@ -60,3 +60,38 @@ import UnexpectedErrorSVG from '@commercetools-frontend/ui-kit/images/maintenanc
 ## Documentation
 
 _coming soon_
+
+## Release
+
+The relase process is _semi-automated_: you only need to **manually** trigger it and [Travis][travis] will take care of the rest.
+
+The steps to prepare and trigger a release are the following:
+
+- ensure to be on the latest `master` branch
+- update the `CHANGELOG.md`
+  - run `yarn changelog` to get a code snippet of the important commits from the last release
+  - copy that and paste it into `CHANGELOG.md` file
+  - make sure that the git tag references are correctly defined and the top entry represents the _new_ tag that you are about to create
+  - add or modify the generated changelog to provide more context about the new release
+- bump the `version` in the `package.json`
+- make a commit and push it to `master` (e.g. `git commit -am "chore: bump version to 2.0.0, update changelog"`)
+  - if you can't push it directly to `master`, open a Pull Request first
+- at this point you can create the `tag`: `git tag -m "Version v2.0.0" v2.0.0`
```

Comment:
Should we rather recommend using `npm version` here? or have [`np`](https://github.com/sindresorhus/np) as part of the deps to run `$(npm bin)/np`? That way we "get more control" and consistency and can also add a `preversion` run script.

Reply:
The publishing step is done by travis, we should not be needing to publish it ourselves (unless for some specific reasons).

Using `npm version` will bump the version, create the commit and tag and push. However, I want to include the changelog in between. So I figured that it's more "safe" to do it step-by-step.

I'm open to suggestions though as long as we achieve that 😇

Reply:
Agreed. Missed that part. Thanks!